src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoParticipantDataDaoTest.java

```diff
@@ -0,0 +1,185 @@
+
+    @Mock
+    DynamoDBMapper mockMapper;
+
+    @Mock
+    PaginatedQueryList<DynamoParticipantData> mockQueryList;
+
+    @Mock
+    QueryResultPage<DynamoParticipantData> mockQueryPage;
+
+    @Captor
+    ArgumentCaptor<DynamoDBQueryExpression<DynamoParticipantData>> queryCaptor;
+
+    @Captor
+    ArgumentCaptor<ParticipantData> participantDataCaptor;
+
+    @Captor
+    ArgumentCaptor<List<DynamoParticipantData>> dataListCaptor;
+
+    @InjectMocks
+    DynamoParticipantDataDao dao;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        MockitoAnnotations.initMocks(this);
+
+        // Create list with a size greater than PageSize
+        participantDataList = ImmutableList.of(
+            createParticipantData("a", "b"),
+            createParticipantData("c", "d"),
+            createParticipantData("e", "f"),
+            createParticipantData("g", "h"),
+            createParticipantData("i", "j"),
+            createParticipantData("k", "l"),
+            createParticipantData("m", "o"),
+            createParticipantData("p", "q"),
+            createParticipantData("r", "s"),
+            createParticipantData("t", "u"),
+            createParticipantData("v", "w")
+        );
+    }
+
+    @Test
+    public void testGetParticipantData() {
+        when(mockMapper.query(eq(DynamoParticipantData.class), any())).thenReturn(mockQueryList);
+        when(mockQueryList.iterator()).thenReturn(participantDataList.iterator());
+        ArgumentCaptor<DynamoDBQueryExpression<DynamoParticipantData>> queryCaptor = ArgumentCaptor.forClass(DynamoDBQueryExpression.class);
+        Condition expectedRangeKeyCondition = new Condition().withComparisonOperator(ComparisonOperator.GE)
+                .withAttributeValueList(new AttributeValue().withS(OFFSET_KEY));
+
+        ForwardCursorPagedResourceList<ParticipantData> result = dao.getParticipantData(USER_ID, OFFSET_KEY, PAGE_SIZE);
+
+        assertEquals(result.getItems(), participantDataList.subList(0, PAGE_SIZE));
+        assertEquals(result.getItems().get(0).getUserId(), USER_ID);
+        assertEquals(result.getNextPageOffsetKey(), result.getItems().get(PAGE_SIZE - 1).getIdentifier());
+        verify(mockMapper).query(any(), queryCaptor.capture());
+        assertEquals(queryCaptor.getValue().getRangeKeyConditions().get("offsetKey"), expectedRangeKeyCondition);
+    }
+
+    //TODO another test with list less than 10 to assert that nextPageOffsetKey is null
+
+    @Test
+    public void testGetParticipantDataNoOffsetKey() {
+        when(mockMapper.query(eq(DynamoParticipantData.class), any())).thenReturn(mockQueryList);
+
+    }
+
+    @Test
+    public void getParticipantDataMultiplePages() {
+
+    }
+
+    @Test
+    public void testGetParticipantDataRecord() {
+    }
+
+    @Test
+    public void testSaveParticipantData() {
+        dao.saveParticipantData(participantData0);
+
+        verify(mockMapper).save(participantDataCaptor.capture());
```

Comment:
You don't need to use a captor. You can just say

```
         verify(mockMapper).save(participantData0);
```

and Mockito will make sure that participantData0 was passed into mockMapper.save().